testnet/inituser.cdc

```diff
@@ -30,6 +30,22 @@ transaction {
             )
         }
 
+        //run the buy without init first
```

Comment:
Why buy? Is that copy pasted from somewhere? I see only initUser changes

Reply:
That is a comment i wrote to explain why it happends

Reply:
Should i remove this comment or not? I do not have insight into what you approve and the client has to match.

Reply:
no it's fine, we will submit for approval now.
I was confused by the wording and would have written something like this myself, but doesn't matter
```suggestion
        // handle the case where buy was ran without init first
```